middleware/validateBody.js

```diff
@@ -0,0 +1,10 @@
+require("express-async-errors");
+
+module.exports =  (validator) => {
```

Comment:
Please, apply the same shorthand consistent here as well. Also be mindful of spacing and its usage.